### PR TITLE
Added test cases for StylesContext

### LINKED ISSUE #232: [Task] Add unit test cases for StylesContext
**Task Description:**
Currently, we run unit tests using jest which can be found [**here**](https://github.com/tjtanjin/react-chatbotify/tree/main/__tests__). 

However, the test cases are not yet comprehensive enough and work can be done to cover more grounds. Specifically for this issue, we hope to add test cases for the `StylesContext`.

**Deliverable(s):**
New unit test cases that cover the testing of the `StylesContext`.

**Additional Context:**
- The `StylesContext` can be found [**here**](https://github.com/tjtanjin/react-chatbotify/tree/main/src/context/StylesContext.tsx)
- An example for `BotRefsContext` can be found [**here**](https://github.com/tjtanjin/react-chatbotify/blob/main/__tests__/context/BotRefsContext.test.tsx) which may serve as a good source for reference

### PR DESCRIPTION
**Pull request by Suvrat1629**
#### Description

Please include a brief summary of the change and include the relevant issue(s).

Closes #232 

#### What change does this PR introduce?
I have added a StylesContext.test.tsx file which contains test code for StylesContext.

Please select the relevant option(s).

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (changes to docs/code comments)

#### What is the proposed approach?

Please give a short overview/explanation on the approach taken to resolve the issue(s).

#### Checklist:

- [x] The commit message follows our adopted [guidelines](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Testing has been done for the change(s) added (for bug fixes/features)
- [ ] Relevant comments/docs have been added/updated (for bug fixes/features)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,12 +50,13 @@
...
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^6.5.0",
+    "@testing-library/jest-dom": "^6.6.2",
     "@testing-library/react": "^16.0.1",
     "@types/dom-speech-recognition": "^0.0.4",
     "@types/jest": "^29.5.13",
     "@types/react": "^18.3.6",
     "@types/react-dom": "^18.3.0",
+    "@types/testing-library__react-hooks": "^3.4.1",
```
Comment (tjtanjin) [inline package.json:59]: This package is deprecated, we don't need to install it.

File: src/components/ChatBotLoader.tsx [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import { Dispatch, MutableRefObject, SetStateAction, useEffect, useMemo } from "react";
-
```
Comment (tjtanjin) [inline src/components/ChatBotLoader.tsx:2]: It's visually convenient to leave a line between react imports and our own imports, so let's keep this space there.

**Review by tjtanjin — CHANGES_REQUESTED** [review]
Hey @Suvrat1629, thank you for the PR. I've left a couple of comments to be addressed. Let me know if you need any clarifications 😊

**Comment by Suvrat1629** [comment]
I have made the changes as you told me please tell me if you need me to make anymore changes

**Comment by tjtanjin** [comment]
> I have made the changes as you told me please tell me if you need me to make anymore changes

Hey @Suvrat1629, there're still outstanding comments unresolved 😅

**Comment by Suvrat1629** [comment]
@tjtanjin updates?

**Review by tjtanjin — APPROVED** [review]
LGTM!

**Pull request merged**